{"trial_id": "0001", "hyperparameters": {"space": [{"class_name": "Int", "config": {"name": "n_hidden", "default": 2, "conditions": [], "min_value": 0, "max_value": 8, "step": 1, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "n_neurons_1", "default": null, "conditions": [], "min_value": 16, "max_value": 256, "step": 1, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "n_neurons_2", "default": null, "conditions": [], "min_value": 16, "max_value": 256, "step": 1, "sampling": "linear"}}, {"class_name": "Float", "config": {"name": "learning_rate", "default": 0.0001, "conditions": [], "min_value": 0.0001, "max_value": 0.01, "step": null, "sampling": "log"}}, {"class_name": "Choice", "config": {"name": "activation", "default": "relu", "conditions": [], "values": ["relu", "tanh"], "ordered": false}}, {"class_name": "Choice", "config": {"name": "optimizer", "default": "sgd", "conditions": [], "values": ["sgd", "adam", "adadelta"], "ordered": false}}, {"class_name": "Float", "config": {"name": "dropout_rate", "default": 0.2, "conditions": [], "min_value": 0.0, "max_value": 0.5, "step": null, "sampling": "linear"}}, {"class_name": "Float", "config": {"name": "l2_lambda", "default": 1e-06, "conditions": [], "min_value": 1e-06, "max_value": 0.01, "step": null, "sampling": "log"}}, {"class_name": "Float", "config": {"name": "momentum", "default": 0.9, "conditions": [], "min_value": 0.0, "max_value": 0.99, "step": null, "sampling": "linear"}}, {"class_name": "Choice", "config": {"name": "initializer", "default": "he_normal", "conditions": [], "values": ["he_normal", "glorot_normal"], "ordered": false}}], "values": {"n_hidden": 8, "n_neurons_1": 195, "n_neurons_2": 69, "learning_rate": 0.0011331682914451785, "activation": "tanh", "optimizer": "adadelta", "dropout_rate": 0.08524696313422081, "l2_lambda": 4.5204328138457374e-05, "momentum": 0.5783044954653371, "initializer": "he_normal", "tuner/epochs": 2, "tuner/initial_epoch": 0, "tuner/bracket": 2, "tuner/round": 0}}, "metrics": {"metrics": {}}, "score": null, "best_step": 0, "status": "FAILED", "message": "Traceback (most recent call last):\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras_tuner/src/engine/base_tuner.py\", line 274, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras_tuner/src/engine/base_tuner.py\", line 239, in _run_and_update_trial\n    results = self.run_trial(trial, *fit_args, **fit_kwargs)\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras_tuner/src/tuners/hyperband.py\", line 427, in run_trial\n    return super().run_trial(trial, *fit_args, **fit_kwargs)\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras_tuner/src/engine/tuner.py\", line 314, in run_trial\n    obj_value = self._build_and_fit_model(trial, *args, **copied_kwargs)\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras_tuner/src/engine/tuner.py\", line 233, in _build_and_fit_model\n    results = self.hypermodel.fit(hp, model, *args, **kwargs)\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras_tuner/src/engine/hypermodel.py\", line 149, in fit\n    return model.fit(*args, **kwargs)\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras/src/utils/traceback_utils.py\", line 70, in error_handler\n    raise e.with_traceback(filtered_tb) from None\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/tensorflow/python/eager/execute.py\", line 53, in quick_execute\n    tensors = pywrap_tfe.TFE_Py_Execute(ctx._handle, device_name, op_name,\ntensorflow.python.framework.errors_impl.InvalidArgumentError: Graph execution error:\n\nDetected at node sparse_categorical_crossentropy/SparseSoftmaxCrossEntropyWithLogits/SparseSoftmaxCrossEntropyWithLogits defined at (most recent call last):\n  File \"/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/runpy.py\", line 197, in _run_module_as_main\n\n  File \"/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/runpy.py\", line 87, in _run_code\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/ipykernel_launcher.py\", line 17, in <module>\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/traitlets/config/application.py\", line 1043, in launch_instance\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/ipykernel/kernelapp.py\", line 725, in start\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/tornado/platform/asyncio.py\", line 195, in start\n\n  File \"/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/asyncio/base_events.py\", line 596, in run_forever\n\n  File \"/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/asyncio/base_events.py\", line 1890, in _run_once\n\n  File \"/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/asyncio/events.py\", line 80, in _run\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/ipykernel/kernelbase.py\", line 513, in dispatch_queue\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/ipykernel/kernelbase.py\", line 502, in process_one\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/ipykernel/kernelbase.py\", line 409, in dispatch_shell\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/ipykernel/kernelbase.py\", line 729, in execute_request\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/ipykernel/ipkernel.py\", line 422, in do_execute\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/ipykernel/zmqshell.py\", line 540, in run_cell\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/IPython/core/interactiveshell.py\", line 3009, in run_cell\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/IPython/core/interactiveshell.py\", line 3064, in _run_cell\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/IPython/core/async_helpers.py\", line 129, in _pseudo_sync_runner\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/IPython/core/interactiveshell.py\", line 3269, in run_cell_async\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/IPython/core/interactiveshell.py\", line 3448, in run_ast_nodes\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/IPython/core/interactiveshell.py\", line 3508, in run_code\n\n  File \"/var/folders/_7/q65t_5bs3653ct73dh3s10w00000gn/T/ipykernel_56636/4059340112.py\", line 1, in <module>\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras_tuner/src/engine/base_tuner.py\", line 234, in search\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras_tuner/src/engine/base_tuner.py\", line 274, in _try_run_and_update_trial\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras_tuner/src/engine/base_tuner.py\", line 239, in _run_and_update_trial\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras_tuner/src/tuners/hyperband.py\", line 427, in run_trial\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras_tuner/src/engine/tuner.py\", line 314, in run_trial\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras_tuner/src/engine/tuner.py\", line 233, in _build_and_fit_model\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras_tuner/src/engine/hypermodel.py\", line 149, in fit\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras/src/utils/traceback_utils.py\", line 65, in error_handler\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras/src/engine/training.py\", line 1807, in fit\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras/src/engine/training.py\", line 1401, in train_function\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras/src/engine/training.py\", line 1384, in step_function\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras/src/engine/training.py\", line 1373, in run_step\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras/src/engine/training.py\", line 1151, in train_step\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras/src/engine/training.py\", line 1209, in compute_loss\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras/src/engine/compile_utils.py\", line 277, in __call__\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras/src/losses.py\", line 143, in __call__\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras/src/losses.py\", line 270, in call\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras/src/losses.py\", line 2454, in sparse_categorical_crossentropy\n\n  File \"/Users/mrbinit/Library/Python/3.9/lib/python/site-packages/keras/src/backend.py\", line 5775, in sparse_categorical_crossentropy\n\nlogits and labels must have the same first dimension, got logits shape [32,10] and labels shape [320]\n\t [[{{node sparse_categorical_crossentropy/SparseSoftmaxCrossEntropyWithLogits/SparseSoftmaxCrossEntropyWithLogits}}]] [Op:__inference_train_function_7492]\n"}